src/types.d.ts

```diff
@@ -13,15 +19,15 @@ declare type ActionInputs = {
 };
 
 declare type ActionOutputs = {
-  /** The number of the issue found, if any. */
-  issue_number: number;
+  /** The number of the issue found, otherwise empty. */
+  issueNumber: number;
 
-  /** Response status. Will be `true` if some issue was found. `false` otherwise. */
-  has_found: boolean;
+  /** Response status. Will be `true` if some issue was found, otherwise `false`. */
+  hasFound: boolean;
 
   /**
-   * Will be `true` if the found issue is closed. `false` otherwise.
-   * The you can use `issue_number` to open it again with [another GitHub Action](https://github.com/marketplace/actions).
+   * Will be `true` if the issue found is closed, otherwise `false`.
+   * Then you can use `issue-number` to open it again with [another GitHub Action](https://github.com/marketplace/actions).
    */
-  is_closed: boolean;
+  isClosed: boolean;
```

Comment:
I'm not sure if I was meant to rename these to kebab-case. I'm not familiar enough with TypeScript to know how your usage of this type differs from `ActionInputs` in `utils.ts`.

As the ActionInputs are using `gh-token` and `gh-repository`, perhaps `githubToken` and `githubRepository` (_the variable names they use in TS_) was the wrong choice and they were meant to match the IDs in `action.yml`?

Until then, I chose to match the ActionOutputs in the same manner, matching the TS variable names.

Reply:
the keys here should match the string passed to `utils.setOutput`

then use `['issue-number']` and so on